modules/ROOT/edits/limitations-self.adoc

```diff
@@ -0,0 +1,19 @@
+= Limitations with Anypoint Runtime Fabric on Self-Managed Kubernetes
```

Comment:
Instead of "Limitations," how about "Requirements for Installing, Configuring, and Managing...."?